tests/features/search.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Search
+
+  Scenario: Search existing Europeana content
+
+    Given I visit a `search page`
```

Comment:
These scenarios are missing a `When` step.

Perhaps the `Given` steps should be `When` steps? Or the `Given` reworded as ``Given I am on a `search page` `` and the first following step made a `When`.